docs/Request.md

```diff
@@ -2,23 +2,62 @@
 
 ## Request
 The first parameter of the handler function is `Request`.<br>
-Request is a core Fastify object containing the following fields:
+Request is a core Fastify object containing the following properties:
 - `query` - the parsed querystring
-- `body` - the body
+- `body` - the body content, properly parsed by the Content-Type value
 - `params` - the params matching the URL
-- `headers` - the headers
+- `headers` - the parsed headers
 - `raw` - the incoming HTTP request from Node core *(you can use the alias `req`)*
-- `id` - the request id
+- `id` - fastify apply an incremental id to every request
```

Comment:
Can you restore the previous description? The incremental id is the default, but in the real world you should use a generator of your own, or use the id that is already present in the request if it is forwarded from another server.

Reply:
You are definitely right! Would you like to contribute to fix it? ;)
https://github.com/fastify/fastify/issues/1035

Reply:
Yes, why not, but it could take me 2-3 weeks, I mean, I will do in my spare time